metals/src/main/scala/scala/meta/internal/metals/debug/DebugProvider.scala

```diff
@@ -792,6 +840,27 @@ class DebugProvider(
 
 object DebugProvider {
 
+  case class ExtendedScalaMainClass(
```

Comment:
This is the same as the original ScalaMainClass from bsp with `shellCommand` added. It is ignored by the BSP server.

Funnily enough putting class in backticks actually worked to serialize the field to `"class"`